codeHF/dpl-config_run5_hf.json

```diff
@@ -761,6 +823,48 @@
             ]
         }
     },
+    "hf-task-bplus": {
+        "cutYCandMax": "1.44",
+        "selectionFlagBPlus": "1",
+        "pTBins": {
+            "values": [
+                "0",
+                "0.5",
+                "1",
+                "2",
+                "3",
+                "4",
+                "5",
+                "7",
+                "10",
+                "13.0",
+                "16.0",
+                "20.0",
+                "24.0"
+            ]
+        }
+    },
+    "hf-task-bplus-mc": {
+        "cutYCandMax": "1.44",
+        "selectionFlagBPlus": "1",
+        "pTBins": {
+            "values": [
+                "0",
+                "0.5",
+                "1",
+                "2",
+                "3",
+                "4",
+                "5",
+                "7",
+                "10",
+                "13.0",
+                "16.0",
+                "20.0",
+                "24.0"
+            ]
+        }
+    },
```

Comment:
This should be removed together with replacing the `--doMC` option by a `processMC` switch ASAP.